python/docs/hashtable/README.md

```diff
@@ -0,0 +1,21 @@
+# Hashtables
+<!-- Short summary or background information -->
+New Data Structure!
+Hash Table is a data structure that is used to store key/value pairs! It uses a hash function to compute an index into a list where the element will be inserted or searched.
+
+## Challenge
+<!-- Description of the challenge -->
+Create your own new data structure: Hash Tables!
+Cover it in tests and make sure it's all in working order for future use.
+
+## Approach & Efficiency
+<!-- What approach did you take? Why? What is the Big O space/time for this approach? -->
+Hash Tables have a time/space complexity of O(1)
```

Comment:
I think O(1) is probably the expected/intended/typical case answer. An interesting thought though is it's possible for the stored keys to hash to the same index. In this case the worst case starts to deviate away from O(1) towards O(n) because will need to traverse potentially the entire linked list of n items. A neat talking point.